crates/sui-swarm-config/tests/snapshots/snapshot_tests__network_config_snapshot_matches.snap

```diff
@@ -1186,4 +1200,3 @@ account_keys:
   - mfPjCoE6SX0Sl84MnmNS/LS+tfPpkn7I8tziuk2g0WM=
   - 5RWlYF22jS9i76zLl8jP2D3D8GC5ht+IP1dWUBGZxi8=
 genesis: "[fake genesis]"
-
```

Comment:
this change should be removed

Reply:
I can remove it but this file has never been touched manually but was auto-generated-updated from running the tests (`cargo test -p sui-swarm-config` etc)